linkedevents/tests/test_api_root_view.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+from events.tests.utils import versioned_reverse as reverse
+
+
+def get_api_root(api_client):
+    url = reverse("api-root")
+
+    return api_client.get(url)
+
+
+@pytest.mark.django_db
+def test_return_correct_routes(api_client):
+    response = get_api_root(api_client)
+    assert response.data["keyword"] == "http://testserver/v1/keyword/"
+    assert response.data["keyword_set"] == "http://testserver/v1/keyword_set/"
+    assert response.data["place"] == "http://testserver/v1/place/"
+    assert response.data["language"] == "http://testserver/v1/language/"
+    assert response.data["organization"] == "http://testserver/v1/organization/"
+    assert response.data["image"] == "http://testserver/v1/image/"
+    assert response.data["event"] == "http://testserver/v1/event/"
+    assert response.data["search"] == "http://testserver/v1/search/"
+    assert response.data["user"] == "http://testserver/v1/user/"
+    assert response.data["registration"] == "http://testserver/v1/registration/"
+    assert response.data["signup"] == "http://testserver/v1/signup/"
```

Comment:
You could add `assert len(response.data) == 11`  here. It will fail when a new api route is added, but at least it needs to be set here then that should the new route be visible or not.

Reply:
Good point. I was just adding an assert for `seats_reservation` route which was added after this PR and added an assert for route length as well.